akka-cluster-sharding/src/multi-jvm/scala/akka/cluster/sharding/ClusterShardingRememberEntitiesSpec.scala

```diff
@@ -73,10 +74,10 @@ abstract class ClusterShardingRememberEntitiesSpecConfig(val mode: String) exten
     }
     """))
 
-  nodeConfig(first, second)(ConfigFactory.parseString(s"""
+  nodeConfig(third)(ConfigFactory.parseString(s"""
     akka.cluster.sharding.distributed-data.durable.lmdb {
-      # use same directory for first and second node (not used at same time)
-      dir = target/ShardingRememberEntitiesSpec/sharding-first-second
+      # use same directory when starting new node on third (not used at same time)
+      dir = target/ShardingRememberEntitiesSpec/sharding-third
```

Comment:
yes, that was from my PR.  The test is assuming that it could use the same directory (lmdb data) for node first and second node, but that will of course not work when it's multi-node.